### DISCUSSION TITLE
pending review

### DISCUSSION THREAD

**Discussion by m2monshi** [WalletConnect/walletconnect-monorepo#2193] [Developer Support]
Hi please take a look of pending review of Project Mondial3d studio
I have been waiting for the approval of my project for six days, should I wait more?

**Comment by 0xBrooke** [comment]
Have you passed the review

**Reply by MikeMat22** [reply]
I've been waiting for 1 month and still nothing

**Reply by glitch-txs** [reply]
hi @MikeMat22 what's the status of your project?

**Comment by fckveza** [comment]
'PENDING REVIEW' waiting too long for something like this.. hahaha.. the V2 update isn't better.. it's actually a pain

**Reply by glitch-txs** [reply]
hey @fckveza what's the name of your project?

**Reply by fckveza** [reply]
Zenpad, @glitch-txs If you are rejected, make a decision.. don't just let it go.. you know the most annoying thing is waiting for something uncertain.. :)

**Reply by glitch-txs** [reply]
Thanks will take a look, we have a lot of submission, sorry for the delay

Also just for clarification, you don't need approval to use the Project ID which comes pre-verified*

**Reply by fckveza** [reply]
a lot because it's left unattended, .. how can small people develop if services like this only help big companies :(, like pancakeSwap, pinksale etc.

**Reply by glitch-txs** [reply]
Approvals are not blockers, you can still use the project ID and everything should work as expected
